htdocs/product/inventory/inventory.php

```diff
@@ -466,6 +466,20 @@ function init_myfunc()
 		} else {
 			print '<a class="classfortooltip" href="#" title="'.dol_escape_htmltag($langs->trans("NotEnoughPermissions")).'">'.$langs->trans("Save").'</a>'."\n";
 		}
+		if ($permissiontoadd && $conf->use_javascript_ajax) {
+			print '&nbsp;';
+			print '<script>';
+			print 'function fillWithExpected(){
+				$(".expectedqty").each(function(){
+				  var object = $(this)[0]
+				  var objecttofill = $("#"+object.id+"_input")[0]
+				  objecttofill.value = object.innerText
+				})
+				console.log("Values filled")
+			  }';
+			print '</script>';
+			print '<a class="butAction" onclick="fillWithExpected()">'.$langs->trans('AutofillWithExpected').'</a>';
```

Comment:
Try to avoid "inline onclick"
Use instead the jquery declaration  
`$( "idofbutton" ).on( "click", fillWithExpected );`
in the js part.
So all js logic is at same place and not embedded with html content.